topics/admin/tutorials/poll-ssa/slides.html

```diff
@@ -0,0 +1,433 @@
+]
+
+
+---
+
+
+### Object Store Backend
+
+
+.pull-left[![Object Store Backend](../../images/poll-ssa/04-Object-Store-Backend.png)]
+
+
+.pull-right[
+- Usually local &/or NAS, a S3.
+]
+
+
+---
+
+
+### Gravity Usage
+
+
+.pull-left[![Gravity Usage](../../images/poll-ssa/05-Gravity-Usage.png)]
+
+
+.pull-right[
+- About 50/50
+- Recent versions seem to include it implicitly
+]
+
+
+---
+
+
+### End-User Support Burden
+
+
+.pull-left[![End User Support Burden](../../images/poll-ssa/06-End-User-Support-Burden.jpg)]
+
+
+.pull-right[
+- 1 hour per week average end-user support
+- Lots of training (though that’s part of the next question)
+- Lots of developing &/or debugging users’ tools/workflows for them.
+]
+
+
+---
+
+
+### User Training Burden
+
+
+.pull-left[![User Training Burden](../../images/poll-ssa/07-User-Training-Burden.jpg)]
+
+
+.pull-right[
+- 1 hour per week average user training
+- Usually makes use of existing Galaxy tutorials, sometimes in-person
+- The tutorials aren’t always specific enough, requiring some custom Q&A
+]
+
+
+---
+
+
+### .pull-right[User Tool Dev & Planemo Usage]
+
+.pull-left[
+.image-50[![User Tool Dev Local](../../images/poll-ssa/08A-User-Tool-Dev-Local.png)]
+.image-50[![User Tool Dev Planemo](../../images/poll-ssa/08B-User-Tool-Dev-Planemo.png)]
+]
+
+
+.pull-right[
+
+ ⠀
+
+ ⠀
+
+ ⠀
+
+- Users developing their own tools is rare, and if they do, they rarely use Planemo, which I found surprising since I thought it was used to simplify tool dev.
+- It turns out tool creation is mostly done through copy/pasting XMLs
+- Planemo shines when it comes to tool testing, linting and publishing
+]
+
+???
+Tried to find a vertical align CSS class, but .center did not work, nor did <div style="vertical-align:middle;"></div>
+So ended up putting an invisible braille character
+
+---
+
+
+### Admin Workflow Dev Burden
+
+
+.pull-left[![Admin Workflow Dev Burden](../../images/poll-ssa/09-Admin-Workflow-Dev-Burden.jpg)]
+
+
+.pull-right[
+Average several hours a month, variable
+]
+
+
+---
+
+
+### Admin Tool Dev Planemo
+
+
+.pull-left[![Admin Tool Dev Planemo](../../images/poll-ssa/10-Admin-Tool-Dev-Planemo.png)]
+
+
+.pull-right[
+Amongst tool-developing admins, 5/7 use Planemo which shows it’s a useful tool for tool testing, linting and publishing.
+]
+
+
+---
+
+
+### Admin Tool Dev Storage
+
+
+.pull-left[![Admin Tool Dev Storage](../../images/poll-ssa/11-Admin-Tool-Dev-Storage.png)]
+
+
+.pull-right[
+- Most tools are files on the same machine as Galaxy
+- Some tools are used from public toolsheds, none from private ones
+- Currently, admins write the tool (managed in git), test & lint it with Planemo, then publish it locally, sometimes versioned (ex: tool_v1.1)
+]
+
+
+---
+
+
+### Admin Tool Dev Creation Burden
+
+
+.pull-left[![Admin Tool Dev Creation Burden](../../images/poll-ssa/12-Admin-Tool-Dev-Creation-Burden.jpg)]
+
+
+.pull-right[
+Small (1 hour per week avg.) amount of tool dev per month.
+]
+
+
```

Comment:
Good idea, will look into this.

Reply:
Added sections and ordered the slides accordingly: 
https://github.com/galaxyproject/training-material/pull/4972/commits/bc1a66ff07ae649a74549a986439f33264994124

I think a further improvement, for all Galaxy training slides in general, would be to implement some kind of automatically  built (optional) progress bar in the footer, like this example (courtes of https://www.slidekit.com/progress-bar-google-slides/ ):
![image](https://github.com/galaxyproject/training-material/assets/11979062/d5b94af8-fb69-4d67-8105-be6889ea3db9)

Reply:
I've moved this into it's own issue, https://github.com/galaxyproject/training-material/issues/5150, I think it's a good suggestion. Perhaps we can construct one automatically from section headings. Or we can reserve some left column space for it, since our slide decks used to be 4:3, not 16:9, so there's some space on the side in most slide decks that could be used for a progress bar à la some beamer themes

![image](https://github.com/galaxyproject/training-material/assets/458683/3a5e26f6-a33f-478f-9349-20d572931979)

Reply:
Good idea with the section headings!
And thanks for the issue.

Also the left idea à la "beamer" (I didn't know what it was called, thanks) makes sense!